Topics/Product_Management/Azure_DevOps.md

```diff
@@ -0,0 +1,24 @@
+## Introduction to Azure DevOps
+Azure DevOps is a product management tool that helps teams manage projects by providing a set of services to support the entire software development life cycle of a product from planning to deployment. For CSC301 teams, two very useful tools are Azure Boards and Azure Test Plans.
+### Azure Boards for task management
+For task management, Azure DevOps provides Azure Board for teams to create work items and assign them to team members. The status of these work items can then be updated during the development phase. 
+A really useful feature of Azure Board is that it allows teams to create work items of different work types and group related work items into a hierarchical structure. Teams can define and customize their own work item types and create a work item hierarchy. A common work item hierarchy that for agile process is:  
+>├── feature 
+>>├── user story 
+>>>├── task
+>>>>├── test case
+>>>>├── bug
+
+Using this set of work item types, we can then group related work items by linking parent work item to children work items. For example, a feature item can be divided into and linked to user stories items.
+Teams can then choose to view all work items in a hierarchical view instead of just a list of work items which really help with project tracking tasks for large projects.
```

Comment:
```suggestion
Teams can then choose to view all work items in a hierarchical view instead of just a list of work items which really helps with tracking tasks for large projects.
```
These changes make more sense to me. What do you think?